erpnext_documentation/www/docs/user/manual/en/accounts/bank-reconciliation.md

```diff
@@ -47,43 +47,60 @@ To view this report, go to **Accounts > Banking and Payments > Bank Reconciliati
 
-If that payment matches, just click on reconcile to reconcile it with this bank transaction.
+You can use Plaid (see [Plaid Integrations page](/docs/user/manual/en/erpnext_integration/plaid_integration)) to automatically synchronize your bank account with ERPNext. All your bank transactions will be automatically imported into ERPNext.
 
-<img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/auto_reconciliation.gif">
+#### 3.2.3 Reconcile the Bank Statement
 
-If ERPNext doesn't propose you any payment, you can always select the corresponding payment manually:
+Once all your bank transactions are imported into ERPNext, you can reconcile them with your existing Vouchers. Go To
+> Accounting > Bank Statement > Bank Reconciliation Tool
 
-<img class="screenshot" alt="Reconcile bank transactions manually" src="{{docs_base_url}}/assets/img/accounts/manual_reconciliation.gif">
+or simply search for 'Bank Reconciliation Tool' in the awesomebar.
 
-You can also create a new payment or invoice directly from the bank reconciliation dashboard.
+1. Select your Company, Bank Account, Bank Statement Start and End Date.
+1. Make sure that the Opening Balance from ERPNext Matches the Opening Balance of your Bank Statement.
+1. Enter the Closing Balance of the Bank Statement.
+1. Saving the Document will create the Tool.
+ <img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/bank_reconciliation_tool.png">
 
-<img class="screenshot" alt="New payment entry" src="{{docs_base_url}}/assets/img/accounts/new_payment.gif">
+1. The Final goal of Bank Reconciliation is to make the Difference amount zero (Green) by either matching to an existing Voucher or creating a new voucher.
+1. For All bank Transactions which are present in the Bank Statement but clearance date is not updated click on the Actions Button you want to Match/ Create Vouchers
+1. For Matching Choose 'Match Against Voucher' in 'Action', The Vouchers that ERPNext found related will be displayed. They will be ranked on the basis of maximum number of fields matched. You can match one or multiple Vouchers against the same Bank Transaction using the checkboxes.
+<img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/match_voucher.png">
+1. To create a new voucher, choose Create Voucher in the Actions and then choose the Document Type. Fill the details that were not available in the Bank Transaction. Clicking on Submit will create the corresponding voucher and update it's clearance date.
+ <img class="screenshot" alt="Reconcile bank transactions" src="{{docs_base_url}}/assets/img/accounts/create_voucher.png">
+1. It is also possible to update the Bank Transaction so that ERPNext can do a better job at finding related Vouchers. Choose Update Bank Transaction in Action, FIll the required details and click on Submit to Save the Bank Transaction.
```

Comment:
It is also possible to update the Bank Transaction so that ERPNext can do a better job at finding related Vouchers. 

- reframe this sentence.